app/views/helpers/_post_form.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= form_with model: @post, url: user_posts_path, scope: :post do |form| %>
+  <div>
+    <%= form.label :title %><br>
+    <%= form.text_field :title %>
+  </div>
+
+  <div>
+    <%= form.label :text %><br>
+    <%= form.text_area :text %><br>
+  </div>
+
+  <div>
+    <%= form.submit %>
+  </div>
+<% end %>
```

Comment:
- [ ] Please put this form's link on the user page so ı user can reach this form.

_Next code reviewer, please check if form work or not._

Reply:
This issue is well resolved! ✔️